qa/standalone/ceph-helpers.sh

```diff
@@ -392,6 +392,17 @@ function test_kill_daemons() {
     teardown $dir || return 1
 }
 
+#
+# return a random TCP port which is not used yet
+#
+# please note, there could be racing if we use this function for
+# a free port, and then try to bind on this port.
+#
+function get_unused_port() {
+    local ip=127.0.0.1
+    python3 -c "import socket; s=socket.socket(); s.bind(('$ip', 0)); print(s.getsockname()[1]); s.close()"
```

Comment:
@tchaikov 
From this I conclude that we are now only using python3??
Or at least for thos that develop?

Reply:
now, i don't think we are only using python3. but i think we should not **rely on** `python` or `python2`, as python2 will retire in 2020. and we are likely to drop python2 support after nautilus.